docker.Makefile

```diff
@@ -9,6 +9,7 @@ LINTER_IMAGE_NAME = docker-cli-lint
 CROSS_IMAGE_NAME = docker-cli-cross
 MOUNTS = -v `pwd`:/go/src/github.com/docker/cli
 VERSION = $(shell cat VERSION)
+GITCOMMIT = $(shell git rev-parse --short HEAD 2> /dev/null || true)
```

Comment:
I don't think we need this. Can we keep the default inside the `scripts/build/.variables` ?

We shouldn't duplicate it.